### PR TITLE
FOP-1360: Add house controller config to U-Boot

### PR DESCRIPTION
**Pull request by 138bit**

### CODE REVIEW

File: board/fancom/whc/MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FANCOM HOUSE CONTROLLER BOARD
+M:	Tim Gruijters <info@nobleo.nl>
+S:	Maintained
+F:	board/fancom/hc/
+F:	include/configs/hc.h
+F:	configs/fancom_hc_defconfig
```
Comment (UberDoefus) [inline board/fancom/whc/MAINTAINERS:6]: fancom_whc_defconfig

File: board/fancom/whc/MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FANCOM HOUSE CONTROLLER BOARD
+M:	Tim Gruijters <info@nobleo.nl>
+S:	Maintained
+F:	board/fancom/hc/
+F:	include/configs/hc.h
```
Comment (UberDoefus) [inline board/fancom/whc/MAINTAINERS:5]: whc.h

File: board/fancom/whc/MAINTAINERS [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FANCOM HOUSE CONTROLLER BOARD
+M:	Tim Gruijters <info@nobleo.nl>
+S:	Maintained
+F:	board/fancom/hc/
```
Comment (UberDoefus) [inline board/fancom/whc/MAINTAINERS:4]: board/fancom/whc/

File: board/fancom/whc/Makefile [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: GPL-2.0
+#
+# (C) Copyright 2012 Stephen Warren
+
+obj-y	:= hc.o
```
Comment (UberDoefus) [inline board/fancom/whc/Makefile:5]: whc.o

**Review by JvanDooren — CHANGES_REQUESTED** [review]
Please address the comments

File: board/fancom/whc/MAINTAINERS (hunk 1 above)
Comment (138bit) [reply board/fancom/whc/MAINTAINERS:6]: done

File: board/fancom/whc/MAINTAINERS (hunk 2 above)
Comment (138bit) [reply board/fancom/whc/MAINTAINERS:5]: done

File: board/fancom/whc/MAINTAINERS (hunk 3 above)
Comment (138bit) [reply board/fancom/whc/MAINTAINERS:4]: done

File: board/fancom/whc/Makefile (hunk 4 above)
Comment (138bit) [reply board/fancom/whc/Makefile:5]: done

File: configs/fancom_whc_defconfig [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+CONFIG_SYS_MMC_ENV_DEV=0
+CONFIG_ENV_SIZE=0x20000
+CONFIG_ENV_OFFSET=0x0
+CONFIG_ENV_SECT_SIZE=0x10000
+CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
+CONFIG_ENV_SIZE_REDUND=0x20000
+CONFIG_ENV_OFFSET_REDUND=0x20000
+CONFIG_AUTOBOOT=y
+CONFIG_AUTOBOOT_KEYED=y
+CONFIG_AUTOBOOT_STOP_STR="ef0780D8"
```
[the remark this answers is not in the document]
Comment (138bit) [reply configs/fancom_whc_defconfig]: done

**Review by JvanDooren — APPROVED** [review]
All comments addressed

**Review by JvanDooren — CHANGES_REQUESTED** [review]
Does not boot

**Review by JvanDooren — APPROVED** [review]
Tested and boots

**Pull request merged**